loops-arrays-objects.js

```diff
@@ -64,6 +71,12 @@ const NOVIEmployees = [
     {firstName: 'Mark', lastName: 'Rensen'},
 ];
 
+for (let i = 0; i < NOVIEmployees.length; i++) {
+    NOVIEmployees[i].email = NOVIEmployees[i].firstName.toLowerCase() + '.' + NOVIEmployees[i].lastName.toLowerCase() + '@novi.nl';
+}
+console.log (NOVIEmployees);
+
```

Comment:
Ziet er netjes uit! Je had hem ook zo kunnen uitwerken: 

for (let i = 0; i < NOVIEmployees.length; i++) {
    const emailaddress = NOVIEmployees[i].firstName + '.' + NOVIEmployees[i].lastName + '@novi.nl';
    NOVIEmployees[i].email = emailaddress.toLowerCase();
}

console.log(NOVIEmployees);

Maar jouw manier werkt ook goed!